.github/workflows/codeguru-reviewer.yml

```diff
@@ -1,6 +1,6 @@
 
 name: Analyze with CodeGuru Reviewer
-on: [pull_request]
```

Comment:
Why? Do you still get comments on PRs?

Reply:
Assuming the IAM Role only works on Pull Requests if the owner of the fork is also allow-listed to assume the IAM role that is used for scanning. Since this is usually not the case, it creates a poor user experience with lots of workflow failures.